patroni/ha.py

```diff
@@ -723,8 +744,10 @@ def _delete_leader(self):
     def release_leader_key_voluntarily(self):
         self._delete_leader()
         self.touch_member()
+        self._released_leader_key_timestamp = time.time()
```

Comment:
Do we want to have a backoff for cases not related to the failure of the `pre_promote` script?